### PR TITLE
[onert/test] Rename tool nnpackage_run to onert_run

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit renames test tool nnpackage_run to onert_run.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue:  #9864

### CODE REVIEW

**Comment by glistening** [comment]
Farewell `nnpackage_run`.

@hseok-oh namespace `nnpkg_run` would be better to be updated. (I think you already know.)

**Comment by hseok-oh** [comment]
>namespace nnpkg_run would be better to be updated. (I think you already know.)

I didn't know. Thanks.

File: infra/nnfw/cmake/CfgOptionFlags.cmake [hunk 1]
```diff
@@ -19,7 +19,7 @@ option(BUILD_RUNTIME_NNAPI_TEST "Build Runtime NN API Generated Test" ON)
 option(BUILD_RUNTIME_NNFW_API_TEST "Build Runtime NNFW API Tests" ON)
 option(BUILD_TFLITE_RUN "Build tflite-run" ON)
 option(BUILD_TFLITE_VANILLA_RUN "Build tflite-vanilla-run" OFF)
-option(BUILD_NNPACKAGE_RUN "Build nnpackge_run" ON)
+option(BUILD_ONERT_RUN "Build nnpackge_run" ON)
```
Comment (glistening) [inline infra/nnfw/cmake/CfgOptionFlags.cmake:22]:
```suggestion
option(BUILD_ONERT_RUN "Build onert_run" ON)
```

File: tests/tools/onert_run/src/formatter.h [hunk 2]
```diff
@@ -42,6 +42,6 @@ class Formatter
 protected:
   nnfw_session *session_;
 };
-} // namespace nnpkg_run
+} // namespace onert_run
 
 #endif // __NNPACKAGE_RUN_FORMATTER_H__
```
Comment (glistening) [inline tests/tools/onert_run/src/formatter.h:47]:
```suggestion
#endif // __ONERT_RUN_FORMATTER_H__
```

**Comment by hseok-oh** [comment]
Step

- Rename binary name
- Rename file/dir
  - `nnpackage_run.cc` -> `onert_run.cc`
  - `tests/tools/nnpackage_run` -> `tests/tools/onert_run`
- Update cmake
  - `NNPACKAGE_RUN_SRCS` -> `ONERT_RUN_SRCS`
  - `BUILD_NNPACKAGE_RUN` -> `BUILD_ONERT_RUN`
- Update source
  - Namespace: `nnpkg_run` -> `onert_run`
  - Guard: `__NNPACKAGE_RUN_` -> `__ONERT_RUN_`

**Comment by hseok-oh** [comment]
Done

**Pull request closed without merging**